apps/passport-client/src/dispatch.ts

```diff
@@ -290,17 +290,17 @@ async function finishLogin(user: User, state: AppState, update: ZuUpdate) {
 
   await addDefaultSubscriptions(identity, state.subscriptions);
 
+  // Save to local storage.
```

Comment:
cc @ichub @artwyman, the change in the execution order of this code makes sure that a new user's storage is saved locally AND on the server before the window.location is updated, the `sync` job is called, and the local blob key is used to download the user's storage from the server.